CMakeLists.txt

```diff
@@ -1259,13 +1259,25 @@ set(PCRE2_CONFIG_VERSION_OUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/pcre2-config-vers
 configure_file(${PCRE2_CONFIG_VERSION_IN} ${PCRE2_CONFIG_VERSION_OUT} @ONLY)
 install(FILES ${PCRE2_CONFIG_OUT} ${PCRE2_CONFIG_VERSION_OUT} DESTINATION cmake)
 
-file(GLOB html ${PROJECT_SOURCE_DIR}/doc/html/*.html)
+file(GLOB html ${PROJECT_SOURCE_DIR}/doc/html/*.html ${PROJECT_SOURCE_DIR}/doc/html/*.txt)
+file(
+  GLOB txts
+  ${PROJECT_SOURCE_DIR}/doc/*.txt
+  AUTHORS.md
+  COPYING
+  ChangeLog
+  LICENCE.md
+  NEWS
+  README
+  SECURITY.md
+)
```

Comment:
Fixed discrepancy between `cmake --install` and `./configure && make install`. They should install the same files ... and they do now.